src/utils.jl

```diff
@@ -19,6 +19,65 @@ cholesky_factor(X::Cholesky) = X.U
 cholesky_factor(X::UpperTriangular) = X
 cholesky_factor(X::LowerTriangular) = X
 
+# TODO: Add `check` as an argument?
+"""
+    cholesky_lower(X)
+
+Return the lower triangular Cholesky factor of `X` as a `Matrix`
+rather than `LowerTriangular`.
+
+!!! note
+    This is a thin wrapper around `cholesky(Hermitian(X)).L`
+    but with a custom `ChainRulesCore.rrule` implementation.
+"""
+cholesky_lower(X::AbstractMatrix) = lower_triangular(parent(cholesky(Hermitian(X)).L))
+function ChainRulesCore.rrule(::typeof(cholesky_lower), X::AbstractMatrix)
+    # TODO: Do we need a special rule for `Hermitian`?
```

Comment:
I don't think so, since this `Hermitian(Hermitian(...))` should just be a no-op :shrug: